devops/resources/user-retirement-archiver.sh

```diff
@@ -19,9 +19,9 @@ assume-role ${ROLE_ARN}
 cd $WORKSPACE/tubular
 pip install -r requirements.txt
 
-# In case this is being run without an explicit END_DATE, default to running with "now"
-if [[ "$END_DATE" == "" ]]; then
-    END_DATE=$(date +%Y-%m-%d)
+# In case this is being run without an explicit END_DATE, default to running with "now" - COOL_OFF_DAYS
+if [[ ! -v END_DATE ]]; then
+    END_DATE=$(date --date "$(date "+%B %d, %Y") -$COOL_OFF_DAYS days" +%y-%m-%d)
```

Comment:
I think you should use ISO formatting instead of manual formatting:
```suggestion
    END_DATE=$(date --iso --date "$(date --iso) - $COOL_OFF_DAYS days")
```
Currently, the output produces a 2-digit year instead of 4-digit.

Reply:
Ah, good catch thanks